### PR TITLE
python3Packages.homeconnect: init at 0.6.3

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change

https://github.com/DavidMStraub/homeconnect

This is a Home Assistant dependency.

@ulrikstrid did all the work to get this modules ready. It's a break-out from #119628.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -10233,6 +10233,12 @@
     githubId = 347983;
     name = "Udo Spallek";
   };
+  ulrikstrid = {
```
Comment (mweinelt) [inline maintainers/maintainer-list.nix:10236]: Creating a maintainer for someone else is questionable at best.

Comment (fabaff) [reply maintainers/maintainer-list.nix:10236]: Are you suggesting to leave the commit in #120532?

Comment (mweinelt) [reply maintainers/maintainer-list.nix:10236]: Not suggesting anything, I just think this is another #120284.

Comment (fabaff) [reply maintainers/maintainer-list.nix]: I remove the commit.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 120532` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.homeconnect</li>
    <li>python39Packages.homeconnect</li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 120532` at db269fc1 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.homeconnect</li>
    <li>python39Packages.homeconnect</li>
  </ul>
</details>

**Review by mweinelt — APPROVED** [review]

**Pull request merged**